Formula/ocrmypdf.rb

```diff
@@ -40,11 +39,6 @@ class Ocrmypdf < Formula
 
   fails_with gcc: "5"
 
-  resource "cffi" do
```

Comment:
I don't see how this can be correct. cryptography depends on cffi.

Reply:
Oh, good point. Dependency chain is ocrmypdf -> pdfminer.six -> cryptography -> cffi -> libffi. 

ocrmypdf doesn't actually use pdfminer.six's cryptography features, but that's too a fine detail for brew to track.

I updated the PR to drop only the leptonica dependency.